ui/src/side_panel.tsx

```diff
@@ -32,33 +32,36 @@ export interface SidePanel {
   items: Component[]
   /** The width of the dialog, e.g. '400px'. Defaults to '600px'. */
   width?: S
+  /** True to prevent closing when clicking or tapping outside the side panel. Prevents interacting with the page behind the side panel. Defaults to False. */
+  blocking?: B
```

Comment:
Preserve the existing attributes order and place any new optional attributes at the end.